chain/chain/src/store.rs

```diff
@@ -2344,6 +2344,7 @@ impl<'a> ChainStoreUpdate<'a> {
         // Now we can proceed to removing the trie state and flat state
         let mut store_update = self.store().store_update();
         for shard_uid in prev_shard_layout.get_shard_uids() {
+            tracing::info!("GC resharding, block_hash {}, shard_uid {}", block_hash, shard_uid);
```

Comment:
Please check out the style guidelines, on tracing use at https://github.com/near/nearcore/blob/master/docs/practices/style.md#Tracing.

```suggestion
            tracing::info!(target: "...", block_hash, shard_uid, "GC resharding");
```

Reply:
Apologies